In musl libc __GLIBC__ is not defined, while it has the
same required header files for VC_MUSIC to work.
Just disable the additional __GLIBC__ shield.

--- a/src/dial.c	2013-05-09 20:32:02.000000000 +0200
+++ b/src/dial.c	2015-06-22 02:44:34.768551564 +0200
@@ -39,11 +39,9 @@
 #include "intl.h"
 
 #ifdef VC_MUSIC
-#  if defined(__GLIBC__)
-#    include <sys/ioctl.h>
-#    include <sys/kd.h>
-#    include <sys/time.h>
-#  endif
+#include <sys/ioctl.h>
+#include <sys/kd.h>
+#include <sys/time.h>
 #endif
 
 enum { CURRENT_VERSION = 6 };
